src/components/dialogs/parameters/voltageinit/equipment-selection-parameters.tsx

```diff
@@ -5,46 +5,37 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { ElementType } from '@gridsuite/commons-ui';
+import { ElementType, EquipmentType } from '@gridsuite/commons-ui';
```

Comment:
Ok but it's not exactly the same enum, there is no TIE_LINE in the commons-ui enum.